monkey/common/agent_plugins/agent_plugin_manifest.py

```diff
@@ -27,6 +27,7 @@ class AgentPluginManifest(InfectionMonkeyBaseModel):
     supported_operating_systems: Tuple[OperatingSystem, ...] = (
         OperatingSystem.WINDOWS,
         OperatingSystem.LINUX,
+        OperatingSystem.ANY,
```

Comment:
I don't think it applies here. If it supports both windows and linux, then it should be [OperatingSystem.WINDOWS, OperatingSystem.LINUX].

Reply:
Read the logic in the issue: https://github.com/guardicore/monkey/issues/2811. I think it makes sense if it is ANY.

Reply:
The default should just be `ANY`